src/devtools/client/debugger/src/actions/breakpoints/modify.ts

```diff
@@ -190,45 +197,17 @@ export function _removeBreakpoint(
   };
 }
 
-export function removeBreakpointAtGeneratedLocation(
-  cx: Context,
-  target: Location
-): UIThunkAction<Promise<void>> {
-  return async (dispatch, getState, { client, ThreadFront }) => {
-    // Remove any breakpoints matching the generated location.
-    const breakpoints = getBreakpointsList(getState());
-    for (const { location } of breakpoints) {
-      if (location.sourceId == target.sourceId && comparePosition(location, target)) {
-        dispatch(removeBreakpoint(location, ThreadFront.recordingId!, cx));
-      }
-    }
-
-    // Remove any remaining pending breakpoints matching the generated location.
-    const pending = getPendingBreakpointList(getState(), ThreadFront.recordingId!);
-    for (const { location } of pending) {
-      if (location.sourceUrl == target.sourceUrl && comparePosition(location, target)) {
-        // @ts-expect-error location mismatch
-        dispatch(removePendingBreakpoint(location, ThreadFront.recordingId!, cx));
-      }
-    }
-
-    // remove breakpoint from the server
-    await client.removeBreakpoint(target);
-  };
-}
-
 export function disableBreakpoint(
   cx: Context,
   initialBreakpoint: Breakpoint
 ): UIThunkAction<Promise<void>> {
-  return async (dispatch, getState, { client, ThreadFront }) => {
-    // @ts-expect-error Breakpoint location mismatch
+  return async (dispatch, getState, { client }) => {
     const breakpoint = getBreakpoint(getState(), initialBreakpoint.location);
     if (!breakpoint || breakpoint.disabled) {
       return;
     }
 
-    dispatch(setBreakpoint({ ...breakpoint, disabled: true }, ThreadFront.recordingId!, cx));
```

Comment:
:speech_balloon: why _did_ we have a `recordingId` here anyway?

Reply:
It was so that we could store it in Pending Breakpoints I think?